src/config.ts

```diff
@@ -1,2 +1,2 @@
-export const defaultVersion = '20220526';
+export const defaultVersion = '20220711';

```

Comment:
@LiamGoLogic  @AdrielA-GoLogic 

Since we have changes on send SMS API request body (tho optional), is there a need to create a new public API version since we have changes on send SMS API request body?

Reply:
> @LiamGoLogic @AdrielA-GoLogic
> 
> Since we have changes on send SMS API request body (tho optional), is there a need to create a new public API version since we have changes on send SMS API request body?

Good question, I wasn't sure about this either. Having checked the versions, in the PublicAPI service, it seems like it is only concerned with the routes, not the contents of the request so I'm guessing there is no need to create a new version. Also brought it up with Mike and he didn't say that I need to create a new version.

Reply:
@juddfsosagologic @LiamGoLogic what we are using right now is 20220711. Is this just an update from master?

Reply:
version 20220711 is set on v1.3.0 branch. But yeah from master